### PR TITLE
God's work

### PR DESCRIPTION
**Pull request by jamerica**

### CODE REVIEW

**Comment by sareg0** [comment]
woah...

**Comment by Hamatti** [comment]
This is great, thanks @jamerica !

File: _posts/2013-05-10-EYDeploy.markdown [hunk 1]
```diff
@@ -8,26 +8,25 @@ permalink: engineyard
 
 *Created by Mary Jenn, [@mfjenn](https://twitter.com/mfjenn)*
 
-#### Ensure You're Using PostGres for Your Database
+#### Ensure You're Using Postgres for Your Database
```
Comment (JuanitoFatas) [inline _posts/2013-05-10-EYDeploy.markdown]: PostgreSQL?

```
Ensure You're Using PostgreSQL for Your Database
```

**Comment by JuanitoFatas** [comment]
I would say maybe change the commit message to be more descriptive? And later if could split into smaller commits / Pull Requests would be great for people to review. Nice work :+1:

**Comment by jamerica** [comment]
@JuanitoFatas Merge.

**Comment by hsbt** [comment]
I couldn't review this largely changes. Please split pull request.

**Comment by alicetragedy** [comment]
@jamerica: please split the pull request and make the commit message more clear as already has been asked above. Thanks!

**Comment by sareg0** [comment]
Is this not some sort of bot?

On Thursday, 14 May 2015, Laura <notifications@github.com> wrote:

> @jamerica <https://github.com/jamerica>: please split the pull request
> and make the commit message more clear as already has been asked above.
> Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/railsgirls/railsgirls.github.com/pull/240#issuecomment-102153704>
> .
>


-- 
Kindest Regards,
Sara and the Rails Girls Summer of Code Team
@RailsGirlsSoC <https://twitter.com/RailsGirlsSoC>

Kindest Regards,
Sara
@sareg0 <https://twitter.com/sareg0>
+49(0)17699393251

**Comment by alicetragedy** [comment]
yep, so I'd like to go ahead and close this PR. A lot of it are irrelevant changes, and I feel that going through all of it and reviewing would be more work than actually creating new pull requests.

**Pull request closed without merging**